projects/components/src/modal/modal.ts

```diff
@@ -9,7 +9,11 @@ export interface ModalConfig<TData = unknown> {
   title?: string;
   data?: TData;
   closeOnEscapeKey?: boolean;
-  styles?: Dictionary<unknown>;
+  styles?: ModalStyles;
+}
+
+export interface ModalStyles {
```

Comment:
I still have reservations about this. Could you please add a todo here? We should come back and discuss this before adding more styles

Reply:
approving for now.

Reply:
@anandtiwary added a todo. please reapprove :)